visualization/plop-templates/service.spec.ts.hbs

```diff
@@ -1,15 +1,25 @@
-import { {{properCase name}}Service } from "./{{camelCase name}}.service";
+import { {{properCase name}}Service } from "./{{camelCase name}}.service"
 
 describe("{{properCase name}}Service", () => {
 
-    let {{camelCase name}}Service: {{properCase name}}Service;
+    let {{camelCase name}}Service: {{properCase name}}Service
 
     beforeEach(function() {
```

Comment:
use arrow-function `() => {}` instead of `function()`

Reply:
guter Einwand, hab ich übersehen